src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1201,130 +1111,122 @@ void runtime::gpu::GPU_Emitter::emit_ReduceWindow(EMIT_ARGS)
 
     const ngraph::op::ReduceWindow* reduce_window_op =
         static_cast<const ngraph::op::ReduceWindow*>(node);
-    writer.block_begin();
+
+    if (out[0].get_size() == 0)
+    {
+        return "";
+    }
+
+    size_t index;
+    // one of args0 axes has zero size, zero output, use args1 value
+    if (args[0].get_size() == 0)
+    {
```

Comment:
those can also be moved to cuda_emitter.
`memset` cuda_emitter has been added in PR #2076.